test/test_bin/test_check_mta_sts.py

```diff
@@ -0,0 +1,103 @@
+import unittest
+import boto3
+from botocore.exceptions import NoCredentialsError
+from moto import mock_s3
```

Comment:
Looks like the [error here](https://github.com/ministryofjustice/operations-engineering/actions/runs/9648814339/job/26610890023#step:7:225) is saying that it's struggling to import `mock_s3` from the `moto` library (potentially it doesn't exist? 🤔)

I've not used moto before, but I would suggest having a read around their [documentation](https://pypi.org/project/moto/) to understand it's basic usage 👀

Reply:
Moto Error has been fixed